packages/mermaid/src/diagrams/quadrant-chart/parser/quadrant.jison

```diff
@@ -104,6 +116,13 @@ statement
 
 points
   : text point_start point_x point_y {yy.addPoint($1, $3, $4);}
+  | text point_start point_x point_y point_radius {yy.addPoint($1, $3, $4, $5);}
+  | text point_start point_x point_y point_color  {yy.addPoint($1, $3, $4, "", $5);}
+  | text point_start point_x point_y stroke_color {yy.addPoint($1, $3, $4, "", "", $5);}
+  | text point_start point_x point_y stroke_width {yy.addPoint($1, $3, $4, "", "", "", $5);}
+  | text point_start point_x point_y point_radius point_color {yy.addPoint($1, $3, $4, $5, $6);}
+  | text point_start point_x point_y point_radius point_color stroke_color {yy.addPoint($1, $3, $4, $5, $6, $7);}
+  | text point_start point_x point_y point_radius point_color stroke_color stroke_width {yy.addPoint($1, $3, $4, $5, $6, $7, $8);}
   ;
```

Comment:
Instead of adding the styling options here, you can accept a styling string here and parse and process it in JS like what's done in flowchart. 

https://github.com/mermaid-js/mermaid/blob/34e0942fa227086bd3db8d68876dd3637dc41e8a/packages/mermaid/src/diagrams/flowchart/parser/flow.jison#L502C1-L539C1